docs/install/macos.md

```diff
@@ -1,4 +1,5 @@
-# How to install Lean and mathlib on MacOS
+How to install Lean and mathlib on MacOS
+===
```

Comment:
What's the point of switching to markdown convention orthogonal to what is used in all other markdown files of this repository?

Reply:
It wasn't typesetting correctly. The title containing backticks doesn't render correctly using the `##` markup.